.github/workflows/build.yml

```diff
@@ -107,11 +107,11 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
     env:
-      # Set up wheels matrix.  This is CPython 3.8--3.10 for all OS targets.
-      CIBW_BUILD: "cp3{8,9,10,11}-*"
+      # Set up wheels matrix.  This is CPython 3.9--3.12 for all OS targets.
+      CIBW_BUILD: "cp3{9,10,11,12}-*"
       # Numpy and SciPy do not supply wheels for i686 or win32 for
       # Python 3.10+, so we skip those:
-      CIBW_SKIP: "*-musllinux* cp3{8,9,10,11}-manylinux_i686 cp3{8,9,10,11}-win32"
+      CIBW_SKIP: "*-musllinux* cp3{10,11,12}-manylinux_i686 cp3{10,11,12}-win32"
```

Comment:
Should we not also skip 3.9 here?

Reply:
I think we should skip the same ones we did before plus the 3.12 ones and minus any we've removed.

Agreed the comment is now inaccurate. Perhaps we can investigate before the next release.